content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](http://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+*Create React App* tidak mengatur bagian *backend* ataupun basis data; namun hanya membuat sebuah *frontend build pipeline*. Sehingga, anda dapat menggunakan aplikasi anda dengan *backend* yang anda inginkan. *Create React App* menggunakan [*Babel*](http://babeljs.io/) dan [*Webpack*](https://webpack.js.org/), tapi anda tidak perlu tahu tentang semua itu.
 
-When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app-) and the [User Guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
+Ketika anda siap untuk melanjutkan ke tahap produksi, menjalankan `npm run build` akan membuat *build* yang teroptimasi dari aplikasi anda dalam *folder* `build`. Anda dapat belajar lebih dalam mengenai *Create React App* dari [README](https://github.com/facebookincubator/create-react-app#create-react-app-) dan [Petunjuk Pengguna](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents) *Create React App*.
 
 ### Next.js {#nextjs}
 
-[Next.js](https://nextjs.org/) is a popular and lightweight framework for **static and server‑rendered applications** built with React. It includes **styling and routing solutions** out of the box, and assumes that you're using [Node.js](https://nodejs.org/) as the server environment.
+[*Next.js*](https://nextjs.org/) adalah *framework* popular dan ringan untuk aplikasi statis dan aplikasi *server-rendered* yang dibangun dengan *React*. Framework ini sudah mengatasi masalah ***styling*** dan ***routing***, dan mengasumsikan bahwa anda menggunakan [Node.js](https://nodejs.org/) sebagai *environment* server anda.
 
-Learn Next.js from [its official guide](https://nextjs.org/learn/).
+Pelajari *Next.js* dari [dokumentasi resminya](https://nextjs.org/learn/).
 
 ### Gatsby {#gatsby}
 
-[Gatsby](https://www.gatsbyjs.org/) is the best way to create **static websites** with React. It lets you use React components, but outputs pre-rendered HTML and CSS to guarantee the fastest load time.
+[Gatsby](https://www.gatsbyjs.org/) adalah cara terbaik untuk membuat *website* statis dengan *React*. *Gatsby* membuat anda dapat menggunakan komponen *React*, namun memberikan keluaran berupa *pre-rendered HTML* dan *CSS* untuk menjamin kecepatan waktu pemuatan.
```

Comment:
Gatsby is fine to be not italicized since it's a name 😄

Reply:
oh, I did so many mistakes. I'll fix them soon. Thank you for review. 😄

Reply:
I've fixed them. Please kindly review it again. 😊